src/network/networkd-radv.c

```diff
@@ -1305,6 +1311,49 @@ int config_parse_router_lifetime(
         return 0;
 }
 
+int config_parse_router_retransmit(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        usec_t usec, *router_retransmit_usec = ASSERT_PTR(data);
+        int r;
+
+        assert(filename);
+        assert(section);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (isempty(rvalue)) {
+                *router_retransmit_usec = 0;
+                return 0;
+        }
+
+        r = parse_sec(rvalue, &usec);
+        if (r < 0) {
+                log_syntax(unit, LOG_WARNING, filename, line, r,
+                           "Failed to parse router Neighbor Solicitation retransmit, ignoring assignment: %s", rvalue);
+                return 0;
+        }
+
+        if (usec != USEC_INFINITY &&
+            DIV_ROUND_UP(usec, USEC_PER_MSEC) > UINT32_MAX) {
+                log_syntax(unit, LOG_WARNING, filename, line, 0,
+                           "Router Neighbor Solicitation retransmit = must be in the range 0...%"PRIu32"ms, ignoring: %s", UINT32_MAX, rvalue);
```

Comment:
Also here, just use the name of the setting, don't invent a textual description.

The value is in seconds, `ms` is a mistake? Also need a space between the number and unit.